sky/packages/sky/lib/src/material/bottom_sheet.dart

```diff
@@ -22,7 +24,7 @@ class _BottomSheet extends StatefulComponent {
   }) : super(key: key);
 
   final Widget child;
-  final PerformanceView performance;
+  final Performance performance;
```

Comment:
You might want to look at Drawer for a pattern here.  The trickiness is that these two objects are both trying to control the performance...